### PR TITLE
Added drop listener to announce changes when moving images in editor

### PR DESCRIPTION
**Pull request by dolbex**
Addressing issue here: [#885](https://github.com/Alex-D/Trumbowyg/issues/885) so that when a user drags an image around the editor it fires the appropriate change event.

### CODE REVIEW

File: dist/trumbowyg.js [hunk 1]
```diff
@@ -653,6 +653,12 @@ Object.defineProperty(jQuery.trumbowyg, 'defaultOptions', {
                         }
                     }
                 })
```
Comment (Alex-D) [inline dist/trumbowyg.js]: You should merge drop and cut listener since they have the same behavior:

```
.on('cut drop', ...
```

**Comment by Alex-D** [comment]
I've done the fix! :)

File: dist/trumbowyg.js (hunk 1 above)
Comment (dolbex) [reply dist/trumbowyg.js]: Will do sir.

**Comment by dolbex** [comment]
Thanks so much Alex.

**Pull request closed without merging**